libs/sdk-backend-bear/src/convertors/fromBackend/ObjRefConverter.ts

```diff
@@ -10,12 +10,12 @@ import { isLocalIdRef, isUriRef, ObjectType, ObjRef, ObjRefInScope } from "@good
  * @param defaultType - type to use it the ref has none specified
  * @internal
  */
-export function fromBearRef(ref: ObjRef, defaultType?: ObjectType): ObjRef {
-    if (isUriRef(ref)) {
-        return ref;
+export function fromBearRef(ref: GdcVisualizationObject.ObjQualifier, defaultType?: ObjectType): ObjRef {
+    if (GdcExecuteAFM.isObjectUriQualifier(ref)) {
```

Comment:
Why you need use this guard ? GdcExecuteAFM.isObjectUriQualifier ? isUriRef is not usable ?

Reply:
Because of typings, that object is same, but check wrong type

Reply:
Ok thanks